### ISSUE TITLE
Installation failing

### ISSUE THREAD

**Issue by dschmitz89** [goepp/aspline#2]
Hey,

installing the package with Rstudio from github fails at the moment.

Error Message:

```
Error in parse_Rd("/tmp/RtmpqbljS6/Rbuild3e7372d5687a/aspline/man/nmr.Rd",  : 
Unexpected end of input (in " quoted string opened at nmr.Rd:15:25)
```

After checking that file, I saw that a string is not closed. However, after adding it, manual building of the package also does not work. I am new to R, so it might be my mistake:

```
package or namespace load failed for ‘aspline’ in namespaceExport(ns, exports):
 undefined exports: sadhanala, spa_het6
```

Do you have a suggestion maybe?

**Comment by bpblakely** [comment]
@dschmitz89 I just got it to work for myself. I closed the string in the nmr.Rd file, just like you did. 

```
\usage{
\code{data(nmr)}
\code{plot(nmr, type = "l")}
}
```

Then I went to the NAMESPACE file in the main directory and removed.

```diff
-export(sadhanala)
-export(spa_het6)
```
It compiled for me afterwards.

If you don't want to do any extra work, then just use ```devtools::install_github("bpblakely/aspline")``` to install this package. This is my own fork that just fixes these 2 small install errors.
